acceptance/lib/puppet/acceptance/rpm_util.rb

```diff
@@ -4,19 +4,31 @@ module RpmUtils
       # Utilities for creating a basic rpm package and using it in tests
       @@defaults = {:repo => '/tmp/rpmrepo', :pkg => 'mypkg', :publisher => 'tstpub.lan', :version => '1.0'}
 
+      def rpm_provider(agent)
+        os = on(agent, facter('operatingsystem')).stdout.chomp
+        maj_version = on(agent, facter('operatingsystemmajrelease')).stdout.chomp
+        if os == 'Fedora' && maj_version.to_i >= 22
```

Comment:
Can we instead just test if the DNF executable exists, so we don't have to update this when other RPM-based distros move over?